lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -494,6 +494,33 @@ public void shouldValidateIssuedAtWithLeeway() throws Exception {
         assertThat(jwt, is(notNullValue()));
     }
 
+	// Issued At with future date
+	@Test (expected = InvalidClaimException.class)
```

Comment:
This test is already present under the name `shouldThrowOnInvalidIssuedAtIfPresent`. Please remove it.

Reply:
done, thank you.